test/run_tests.sh

```diff
@@ -247,6 +247,7 @@ function run_xla_op_tests3 {
   run_test "$CDIR/spmd/test_xla_auto_sharding.py"
   run_test "$CDIR/spmd/test_spmd_parameter_wrapping.py"
   run_test "$CDIR/spmd/test_mp_input_sharding.py"
+  run_test "$CDIR/spmd/test_spmd_lowering_context.py"
```

Comment:
I learned that we also need to add this test to `test/tpu/run_tests.sh`. Otherwise it won't be exercised in TPU CI, which IIUC is the only CI that exercises SPMD.

Reply:
Nice, done.